index.js

```diff
@@ -1,92 +1,82 @@
-// ***** Question 1 *****
-// *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
-
+// Q1
+let counter = 1
+counter = 2
 // console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
-
-
-// ***** Question 2 *****
-// *** Uncomment the lines below to test
-// *** after testing, comment the line causing the error back in 
-// *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
 
-// console.log(name)
-// => "Raffy"
-// name = "Not Raffy" 
-// => TypeError
-// console.log("%c----------", "color: red") 
 
 
+// Q2
+const name = "Ben"
+//name = "Not Raffy" 
 
-// ***** Question 3 *****
-// *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
 
-// console.log(drinkWater(12))
-// // => "Man I sure am thirsty"
-// // => "Ahh that hits the spot"
-// // => 11
-// console.log("%c----------", "color: red") 
 
+// Q3
+function drinkWater(currentThirstLevel) {
+  console.log("Man I sure am thirsty")
+  currentThirstLevel -= 1
+  console.log("Ahh that hits the spot")
+  return currentThirstLevel
+}
+//console.log(drinkWater(12))
```

Comment:
Nice work. remembering that we need to explicitly return values in JS.

As an additional exercise, how would this look as an arrow function?